.github/workflows/android_release.yml

```
@@ -0,0 +1,30 @@
+        java-version: '17'
+        distribution: 'temurin'
+        cache: gradle
+
+    - name: Grant execute permission for gradlew
+      run: chmod +x androidApp/gradlew
+
+    - name: Build with Gradle
+      run: cd androidApp && ./gradlew assembleDebug
+
+    - name: Upload APK
+      uses: actions/upload-artifact@v3
```

actionlint (1.7.10)

the runner of "actions/upload-artifact@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)